libs/utils/utils.js

```diff
@@ -411,6 +411,20 @@ export function decorateSVG(a) {
   }
 }
 
+export function decorateVideo(a) {
+  const { pathname, hash } = a;
+  if (!pathname.match('media_.*.mp4$')) return;
+
+  const isAutoplay = !!(hash?.includes('autoplay'));
+
+  const attrs = isAutoplay ? 'playsinline autoplay loop muted' : 'playsinline controls';
+  const video = `<video ${attrs}>
+        <source src=".${pathname}" type="video/mp4" />
+      </video>`;
```

Comment:
Not sure, but I think `createTag` is typically preferred, though I understand with `${attrs}` and `${pathname}`this might be the more performant way.

Reply:
Yeah, video `attributes` was not working properly since `createTag` takes object for attributes. Also, it's much cleaner in this situation.